docs/quick-start/starknet-devnet-with-asdf.mdx

```diff
@@ -0,0 +1,54 @@
+# StarkNet Devnet - Installation and Setup with asdf
+
+This document provides a guide to install and run **StarkNet Devnet** using `asdf`, a Rust-based tool for developers. Below, you'll find instructions for installation using `asdf`.
```

Comment:
Hi, thanks for the contribution @cidkagenow , you should change "StarkNet" to "Starknet" in all lines.